### ISSUE TITLE
 UndefVarError: series not defined when call unicodeplots bar

### ISSUE THREAD

**Issue by nanjj** [JuliaPlots/Plots.jl#1904]
```
unicodeplots()
Plots.UnicodePlotsBackend()

julia> bar(randn(999))
Error showing value of type Plots.Plot{Plots.UnicodePlotsBackend}:
ERROR: UndefVarError: series not defined
Stacktrace:
 [1] addUnicodeSeries!(::UnicodePlots.Plot{UnicodePlots.BrailleCanvas}, ::Dict{Symbol,Any}, ::Bool, ::Array{Float64,1}, ::Array{Float64,1}) at /Users/nanjj/.julia/packages/Plots/EJpx2/src/backends/unicodeplots.jl:141
 [2] rebuildUnicodePlot!(::Plots.Plot{Plots.UnicodePlotsBackend}, ::Int64, ::Int64) at /Users/nanjj/.julia/packages/Plots/EJpx2/src/backends/unicodeplots.jl:107
 [3] unicodeplots_rebuild(::Plots.Plot{Plots.UnicodePlotsBackend}) at /Users/nanjj/.julia/packages/Plots/EJpx2/src/backends/unicodeplots.jl:190
 [4] _display(::Plots.Plot{Plots.UnicodePlotsBackend}) at /Users/nanjj/.julia/packages/Plots/EJpx2/src/backends/unicodeplots.jl:201
 [5] display(::Plots.PlotsDisplay, ::Plots.Plot{Plots.UnicodePlotsBackend}) at /Users/nanjj/.julia/packages/Plots/EJpx2/src/output.jl:145
 [6] display(::Any) at ./multimedia.jl:287
 [7] #invokelatest#1 at ./essentials.jl:742 [inlined]
 [8] invokelatest at ./essentials.jl:741 [inlined]
 [9] print_response(::IO, ::Any, ::Any, ::Bool, ::Bool, ::Any) at /Users/osx/buildbot/slave/package_osx64/build/usr/share/julia/stdlib/v1.1/REPL/src/REPL.jl:155
 [10] print_response(::REPL.AbstractREPL, ::Any, ::Any, ::Bool, ::Bool) at /Users/osx/buildbot/slave/package_osx64/build/usr/share/julia/stdlib/v1.1/REPL/src/REPL.jl:140
 [11] (::getfield(REPL, Symbol("#do_respond#38")){Bool,getfield(REPL, Symbol("##48#57")){REPL.LineEditREPL,REPL.REPLHistoryProvider},REPL.LineEditREPL,REPL.LineEdit.Prompt})(::Any, ::Any, ::Any) at /Users/osx/buildbot/slave/package_osx64/build/usr/share/julia/stdlib/v1.1/REPL/src/REPL.jl:714
 [12] #invokelatest#1 at ./essentials.jl:742 [inlined]
 [13] invokelatest at ./essentials.jl:741 [inlined]
 [14] run_interface(::REPL.Terminals.TextTerminal, ::REPL.LineEdit.ModalInterface, ::REPL.LineEdit.MIState) at /Users/osx/buildbot/slave/package_osx64/build/usr/share/julia/stdlib/v1.1/REPL/src/LineEdit.jl:2273
 [15] run_frontend(::REPL.LineEditREPL, ::REPL.REPLBackendRef) at /Users/osx/buildbot/slave/package_osx64/build/usr/share/julia/stdlib/v1.1/REPL/src/REPL.jl:1035
 [16] run_repl(::REPL.AbstractREPL, ::Any) at /Users/osx/buildbot/slave/package_osx64/build/usr/share/julia/stdlib/v1.1/REPL/src/REPL.jl:192
 [17] (::getfield(Base, Symbol("##734#736")){Bool,Bool,Bool,Bool})(::Module) at ./client.jl:362
 [18] #invokelatest#1 at ./essentials.jl:742 [inlined]
 [19] invokelatest at ./essentials.jl:741 [inlined]
 [20] run_main_repl(::Bool, ::Bool, ::Bool, ::Bool, ::Bool) at ./client.jl:346
 [21] exec_options(::Base.JLOptions) at ./client.jl:284
 [22] _start() at ./client.jl:436
```

**Comment by mkborregaard** [comment]
I can't replicate - can you paste `versioninfo()` and `]status`?

**Comment by o01eg** [comment]
```julia
julia> bar(randn(999))
Error showing value of type Plots.Plot{Plots.UnicodePlotsBackend}:
ERROR: UndefVarError: series not defined
Stacktrace:
 [1] addUnicodeSeries!(::UnicodePlots.Plot{UnicodePlots.BrailleCanvas}, ::Dict{Symbol,Any}, ::Bool, ::Array{Float64,1}, ::Array{Float64,1}) at /home/o01eg/.julia/packages/Plots/UQI78/src/backends/unicodeplots.jl:119
 [2] rebuildUnicodePlot!(::Plots.Plot{Plots.UnicodePlotsBackend}, ::Int64, ::Int64) at /home/o01eg/.julia/packages/Plots/UQI78/src/backends/unicodeplots.jl:85
 [3] unicodeplots_rebuild(::Plots.Plot{Plots.UnicodePlotsBackend}) at /home/o01eg/.julia/packages/Plots/UQI78/src/backends/unicodeplots.jl:168
 [4] _display(::Plots.Plot{Plots.UnicodePlotsBackend}) at /home/o01eg/.julia/packages/Plots/UQI78/src/backends/unicodeplots.jl:179
 [5] display(::Plots.PlotsDisplay, ::Plots.Plot{Plots.UnicodePlotsBackend}) at /home/o01eg/.julia/packages/Plots/UQI78/src/output.jl:145
 [6] (::getfield(REPL, Symbol("#do_respond#38")){Bool,getfield(REPL, Symbol("##48#57")){REPL.LineEditREPL,REPL.REPLHistoryProvider},REPL.LineEditREPL,REPL.LineEdit.Prompt})(::Any, ::Any, ::Any) at /tmp/portage/dev-lang/julia-1.1.0/work/julia-1.1.0/usr/share/julia/stdlib/v1.1/REPL/src/REPL.jl:714
julia> versioninfo()
Julia Version 1.1.0
Commit 503ed46* (2019-02-09 07:43 UTC)
Platform Info:
  OS: Linux (x86_64-pc-linux-gnu)
  CPU: Intel(R) Core(TM) i7-6700K CPU @ 4.00GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-6.0.1 (ORCJIT, skylake)
(v1.1) pkg> status
    Status `~/.julia/environments/v1.1/Project.toml`
  [91a5bcdd] Plots v0.23.0
  [b8865327] UnicodePlots v1.1.0
```

**Comment by o01eg** [comment]
It was added in https://github.com/JuliaPlots/Plots.jl/commit/9a5f42c251e48f356ad38335725341b2679a3e6f#diff-05846e862e4d1ac1639abfb015c3e600 but nothing defines `series` variable

**Comment by o01eg** [comment]
Current test cann't catch this error because it doesn't `dispay` plot. I suppose it should be something like
```julia
@test isa(display(plot(rand(10))), Nothing) == true
@test isa(display(bar(randn(10))), Nothing) == true
```

**Comment by mkborregaard** [comment]
Closed in #1915
